---
geometry_msgs/Point[] coordinates
bool success